### PR TITLE
Pasos de registro

### PR DESCRIPTION
**Pull request by JosephSC0121**
# Page Register 

## Description
Creacion pagina de registro
### Summary
paso 2
image
paso 3
image

## How Has This Been Tested?
Se testeo directamente en la web

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/modules/auth/templates/register-step-2.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                    <FiArrowLeftCircle className="h-7 w-7" />
+                </Link>
+                <span> Vuelve al paso uno </span>
+            </div>
+            <span className="font-bold text-3xl m-6">Paso 2: Escoge tu Rol</span>
+            <div className="flex flex-col m-6">
+                <span className="text-lg">Una vez lo escojas, no podrás cambiarlo.</span>
+            </div>
+            <div className="flex flex-col sm:flex-row justify-center items-center gap-4 sm:gap-8 m-6">
+                <ButtonCard title="Asesor" icon={true} description="Si te encargas de revisar documentos" className={""} />
```
Comment (Camilo-Camargo) [inline src/modules/auth/templates/register-step-2.tsx:22]: button card aplicar gap por dentro de elementos

File: src/modules/auth/templates/signup.tsx [hunk 2]
```diff
@@ -1,10 +1,77 @@
 "use client";
 
+import { useFormState } from "react-dom";
+import { authenticate } from "../services/actions";
+import { usePathname } from "next/navigation";
+import { Input } from "@/modules/common/components/input";
+import Link from "next/link";
+import { SubmitButton } from "../../common/components/submit-button";
+
```
Comment (Camilo-Camargo) [inline src/modules/auth/templates/signup.tsx:9]:
```suggestion
```

File: src/modules/auth/types/auth.ts [hunk 3]
```diff
@@ -9,5 +9,8 @@ export enum ROUTES_AUTH {
     SING_UP = "/signUp",
     PRINCIPAL = "/principal",
     HOME = "/home",
-    RESET_PASSWORD = "/reset-password"
+    RESET_PASSWORD = "/reset-password",
+    STEP1 = "/signUp/stepOne",
+    STEP2 = "/signUp/stepTwo",
+    STEP3 = "/signUp/stepThree",
```
Comment (Camilo-Camargo) [inline src/modules/auth/types/auth.ts:15]:
```suggestion
    STEP1 = "/sign-up/step-one",
    STEP2 = "/sign-up/step-two",
    STEP3 = "/sign-up/step-three",
```

File: src/modules/auth/templates/register-step-2.tsx (hunk 1 above)
Comment (JosephSC0121) [reply src/modules/auth/templates/register-step-2.tsx:22]: https://github.com/OwlByTech/optitech-web/pull/11/commits/dcd1685654d2277d71d087aa2d9c976c63999bbe

**Review by Alejandro-Suarez-Rincon — APPROVED** [review]

**Review by CamiloArias12 — APPROVED** [review]

**Pull request merged**